demo/Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM node:8.16.0-stretch AS texture
+
+ARG UNAME=testuser
+ARG UID=1000
+ARG GID=1000
+RUN groupadd -g $GID -o $UNAME
```

Comment:
Usernames are in `/etc/passwd` and specific to each container, while user ids are global to a kernel. Therefore you may not need the username as an argument, only user and group id.

Reply:
(using a standard username)

Reply:
Yes, true, it's more like a nice touch to use the same name inside the container.